### PR TITLE
New protocol/setup input plate

### PR DESCRIPTION
**Pull request by chr1sgrant**

### CODE REVIEW

**Comment by chr1sgrant** [comment]
@synthacebot test this please

File: an/Utility/Setup_Input_Plate/Setup_Input_Plate.an [hunk 1]
```diff
@@ -0,0 +1,213 @@
...
+// Place golang packages to import here
+import (
+	"github.com/antha-lang/antha/antha/anthalib/wtype"
+	"fmt"
+)
+
+// Parameters to this protocol
+Parameters (
+	// optional parameter to specify the well positions for a particular solution
+	// This will conflict with ByRow if both are used together so an error will be returned in that situation.
```
Comment (n-scholes) [inline an/Utility/Setup_Input_Plate/Setup_Input_Plate.an]: Each solution is assigned an array of positions e.g. "water": ["A1","B2"], will add water to wells A1 and B2.

File: an/Utility/Setup_Input_Plate/Setup_Input_Plate.an [hunk 2]
```diff
@@ -0,0 +1,213 @@
...
+	SpecifyWellPositions map[string][]string
+	
+	// If selected the solutions will be added by row (A1, A2, A3 ...), the default is by column (A1, B1, C1 ...).
+	// This will conflict with the SpecifyWellPositions if both are used together so an error will be returned in that situation.
```
Comment (n-scholes) [inline an/Utility/Setup_Input_Plate/Setup_Input_Plate.an]: ByRow cannot be set to true if SpecifyWellPositions contains well declarations. An error will be returned in that situation.

**Comment by chr1sgrant** [comment]
This is fixed now so I'll merge in to test in the UI

**Pull request merged**